### ISSUE TITLE
Error with .torrent

### ISSUE THREAD

**Issue by Lunik** [jpillora/cloud-torrent#16]
When I put the torrent URL into the search bar and click on "Start Torrent", the app return an alert.

## Alert 
![Alert error](http://puu.sh/kwWU5/50c074a60b.png)

## Console Logs
![ConsoleLogs](http://puu.sh/kwX0M/d8734569e5.png)

The .torrent links comme from [Cpasbien.pw](http://www.cpasbien.pw/)

**Comment by jpillora** [comment]
I think its just an invalid torrent, for example http://releases.ubuntu.com/15.04/ubuntu-15.04-desktop-amd64.iso.torrent works

**Comment by Lunik** [comment]
You are right, 
But what I don't understand is that those torrent work fine with a classical uTorrent desktop app.

**Comment by jpillora** [comment]
I'm using https://github.com/anacrolix/torrent underneath and I guess @anacrolix is strictly parsing torrent files. Ping @anacrolix - Hey Matt, are you open to making your torrent parser more lenient?

**Comment by anacrolix** [comment]
Can you provide a sample .torrent?

**Comment by Lunik** [comment]
http://www.cpasbien.pw/telechargement/cv-facile-cvitae.torrent

**Comment by anacrolix** [comment]
I don't think the problem is with the torrent library. That site seems to dislike programmatic retrieval of torrent files. I suspect there's a HTTP error involved between cloud-torrent and cpasbien.pw.

**Comment by anacrolix** [comment]
$ godo github.com/anacrolix/torrent/cmd/torrent ../cv-facile-cvitae.torrent --listen-addr=:50008
2015/10/04 00:15:51 envpprof.go:70: envpprof serving http://127.0.0.1:6064
2015/10/04 00:16:01 main.go:136: downloaded ALL the torrents

Downloads fine, took about ~10s. ~2.8MB torrent.
$ md5sum SetupCVitae.exe 
739422d2267ec06b2b65406203ae4dd1  SetupCVitae.exe

**Comment by Lunik** [comment]
Thanks for the explanation. 
So is there an other way to use this .torrent ?? I haven't seen a button to upload .torrent file...

**Comment by jpillora** [comment]
Yeah true, sorry for the false alarm @anacrolix. Yep so it's just a problem with that site. Downloading from the browser seems to work, though doing a programmatic download doesn't:

```
$ curl -I http://www.cpasbien.pw/telechargement/cv-facile-cvitae.torrent
HTTP/1.1 503 Service Temporarily Unavailable
```

@Lunik Upload torrent is not yet implemented, though I'll modify this issue to be a feature request.

**Comment by Lunik** [comment]
Thanks for your quick answers. 
And nice work @jpillora !!

**Comment by PavilionVI** [comment]
Hi,
I've noticed that private torrents don't work too! When I loaded a torrent from bakabt.me, it kept loading but the download didn't start, on the other hand, I managed to download the same torrent on filestream.me.

here's the torrent I used: https://bakabt.me/download/180332/1453999514/dd8249760304a541d8c8a85cc62a3d11/1867873/military-480p-horriblesubs.torrent

P.S. Thank you for your awesome script.

**Comment by jaswinder-singh** [comment]
try this >> http://kaffeine.herokuapp.com for the sleep problem
